musicrecs/views.py

```diff
@@ -81,10 +81,15 @@ def round(long_id):
                 round.music_type.name, submission.spotify_link
             ) for submission in submission_list
         }
+        if round.snoozin_rec_type == 0:
+            snoozinrt = 'Random'
+        else:
+            snoozinrt = 'Similar'
```

Comment:
Just so you know, another way of doing this would have been `snoozinrt = round.snoozin_rec_type.name`

Reply:
Switched to using this approach. Much cleaner!